test-framework/core/src/main/java/org/keycloak/testframework/events/AdminEventAssertion.java

```diff
@@ -77,6 +77,11 @@ public AdminEventAssertion resourcePath(String... expectedResourcePath) {
         return this;
     }
 
+    public AdminEventAssertion isUUID(String item) {
+        Assertions.assertTrue(36 == item.length() && item.charAt(8) == '-' && item.charAt(13) == '-' && item.charAt(18) == '-' && item.charAt(23) == '-');
+        return this;
+    }
+
```

Comment:
This doesn't make much sense; for checking if id is a UUID that may be something we can just add directly to assertSuccess/Failure.